banzai_floyds_ui/gui/urls.py

```diff
@@ -15,13 +15,14 @@
 """
 from django.contrib import admin
 from django.urls import path, include
-from banzai_floyds_ui.gui.views import banzai_floyds_view, login_view, logout_view
+from banzai_floyds_ui.gui.views import banzai_floyds_view, login_view, logout_view, status_view
 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api', include('banzai_floyds_ui.api.urls')),
     path('', banzai_floyds_view, name="banzai-floyds"),
+    path('status', status_view, name="status"),
```

Comment:
I added a status endpoint that returns simple json as a liveness probe for the deployment.